packages/api/v1/events/events.ts

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+import {getAllEvents, getEventById, updateEventById, deleteEventById, createEvent} from "../../src/controllers/eventController";
+
+const eventRoutes = Router();
+
+eventRoutes.get('/event', getAllEvents);
+eventRoutes.post('/event', createEvent)
+eventRoutes.get('/event/:id', getEventById);
+eventRoutes.put('/event/:id', updateEventById);
+eventRoutes.delete('/event/:id', deleteEventById);
```

Comment:
The endpoint should be `/events` (plural) and should include the api version. AKA it should be `/api/v1/events`.

Reply:
Fixed to be named "api/v1/events"